src/core/ext/filters/client_channel/resolver/polling_resolver.h

```diff
@@ -116,6 +110,12 @@ class PollingResolver : public Resolver {
     kReresolutionRequestedWhileCallbackWasPending,
   };
   ResultStatusState result_status_state_ = ResultStatusState::kNone;
+  /// next resolution timer
+  absl::optional<grpc_event_engine::experimental::EventEngine::TaskHandle>
+      next_resolution_timer_handle_{absl::nullopt};
+  // a raw pointer will suffice since the shared_ptr in channel_args_ will keep
+  // the EventEngine alive
+  grpc_event_engine::experimental::EventEngine* event_engine_;
```

Comment:
This pointer probably isn't necessary.  We can probably just grab it out of `channel_args_` each time we need it.  The code here is not performance-sensitive, so it's probably better to avoid wasting memory than to optimize this access.